<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuType">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionPolygon"/>
    <addaction name="actionFree"/>
    <addaction name="actionText"/>
    <addaction name="separator"/>
    <addaction name="actionEraser"/>
    <addaction name="separator"/>
    <addaction name="actionCancel"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menuLineStyle">
     <property name="title">
      <string>线条风格</string>
     </property>
     <addaction name="actionNoPen"/>
     <addaction name="actionSolidLine"/>
     <addaction name="actionDashLine"/>
     <addaction name="actionDotLine"/>
     <addaction name="actionDashDotLine"/>
     <addaction name="actionDashDotDotLine"/>
    </widget>
    <widget class="QMenu" name="menuBrushStyle">
     <property name="title">
      <string>填充风格</string>
     </property>
     <addaction name="actionNoBrush"/>
     <addaction name="actionSolidPattern"/>
     <addaction name="actionDense1"/>
     <addaction name="actionDense2"/>
     <addaction name="actionDense3"/>
     <addaction name="actionDense4"/>
     <addaction name="actionDense5"/>
     <addaction name="actionDense6"/>
     <addaction name="actionDense7"/>
     <addaction name="actionHorizontal"/>
     <addaction name="actionVertical"/>
     <addaction name="actionCross"/>
     <addaction name="actionForward"/>
     <addaction name="actionBackward"/>
     <addaction name="actionDiagCross"/>
    </widget>
    <addaction name="actionFont"/>
    <addaction name="separator"/>
    <addaction name="actionLineWidth"/>
    <addaction name="actionLineColor"/>
    <addaction name="menuLineStyle"/>
    <addaction name="separator"/>
    <addaction name="actionBrushColor"/>
    <addaction name="menuBrushStyle"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout_Me"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuType"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLine"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionPolygon"/>
   <addaction name="actionFree"/>
   <addaction name="actionText"/>
   <addaction name="separator"/>
   <addaction name="actionEraser"/>
   <addaction name="separator"/>
   <addaction name="actionCancel"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionSave_As"/>
  </widget>
  <action name="actionAbout_Me">
   <property name="text">
    <string>关于我</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>关于QT</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>线段</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>椭圆</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>多边形</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionFree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自由曲线</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>文本</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>清空</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>橡皮擦</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>字体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionLineWidth">
   <property name="text">
    <string>线宽</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionLineColor">
   <property name="text">
    <string>线条颜色</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionBrushColor">
   <property name="text">
    <string>填充颜色</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSolidLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SolidLine</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionDashLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DashLine</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionDotLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DotLine</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionDashDotLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DashDotLine</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionNoPen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NoPen</string>
   </property>
   <property name="toolTip">
    <string>NoPen</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </action>
  <action name="actionDashDotDotLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DashDotDotLine</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionSolidPattern">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SolidPattern</string>
   </property>
   <property name="shortcut">
    <string>Alt+5</string>
   </property>
  </action>
  <action name="actionDense1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionDense2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionDense3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense3</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionDense4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense4</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionDense5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense5</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionDense6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense6</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="actionDense7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense7</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+7</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+9</string>
   </property>
  </action>
  <action name="actionCross">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cross</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionBackward">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backward</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionDiagCross">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DiagCross</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionNoBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NoBrush</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
